extensions/emmet/src/abbreviationActions.ts

```diff
@@ -616,14 +616,30 @@ function expandAbbr(input: ExpandAbbreviationInput): string | undefined {
 				}
 
 				// If wrapping with a block element, insert newline in the text to wrap.
-				if (wrappingNode && inlineElements.indexOf(wrappingNode.name) === -1) {
-					wrappingNode.value = '\n\t' + wrappingNode.value + '\n';
+				if (wrappingNode) {
+					if (inlineElements.indexOf(wrappingNode.name) === -1) {
```

Comment:
Can you pull in the comment on line 618 so that it is above this `if` statement?